third_party/move/move-compiler-v2/tests/checking/inlining/resources_invalid.exp

```diff
@@ -1,20 +1,12 @@
-// -- Model dump before bytecode pipeline
-module 0x42::objects {
-    struct ReaderRef {
-        addr: address,
-    }
-} // end 0x42::objects
-module 0x42::token {
-    use 0x42::objects as obj; // resolved as: 0x42::objects
-    struct Token {
-        val: u64,
-    }
-    public fun get_value(ref: &objects::ReaderRef<token::Token>): u64
-        acquires token::Token(*)
-     {
-        select token::Token.val<&token::Token>({
-          let (ref: &objects::ReaderRef<token::Token>): (&objects::ReaderRef<token::Token>) = Tuple(ref);
-          BorrowGlobal(Immutable)<token::Token>(select objects::ReaderRef.addr<&objects::ReaderRef<token::Token>>(ref))
-        })
-    }
-} // end 0x42::token
+
```

Comment:
Could we add more inline tests, specifically, inline functions that do each of the restricted cases? We may then want to improve error messages for the inlined cases (like is done for visibility of calls).

Reply:
done